src/Chapter06.Tests/Listing06.37.UsingTheMaybeNullAttribute.Tests.cs

```diff
@@ -1,28 +1,27 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_37.Tests
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_37.Tests;
+
+[TestClass]
+public class ProgramTests
 {
-    [TestClass]
-    public class ProgramTests
+    [TestMethod]
+    public void GetObject_CollectionWithItem_ReturnItem()
     {
-        [TestMethod]
-        public void GetObject_CollectionWithItem_ReturnItem()
-        {
-            NullabilityAttributesExamined.GetObject(
-                new string[] { "item" }, item => item is object);
-        }
+        NullabilityAttributesExamined.GetObject(
+            new string[] { "item" }, item => item is object);
```

Comment:
Assert-less test should at least have a comment. Should this be check the return is not null?